parsers/getter/brent.py

```diff
@@ -24,7 +27,8 @@ def make_url(access_key=EIA_ACCESS_KEY):
 def fetch(url):
     r = requests.get(url)
     return r.text
-    return json.loads(r.text)
+    # EP: why json.loads(r.text) will not execute? 
```

Comment:
@epogrebnyak Method will make exit after first return. Will be executed only first one